### PR TITLE
[BREAKING] Require Node.js version >= 14

### PR DESCRIPTION
**Pull request by tobiasso85**
Support for older Node.js releases has been dropped.
Increases the minimum node version required to 14

Node.js v12 eol is 2022-04-30

Also latest yargs  version requires minimum node version 10.
https://github.com/yargs/yargs/blob/master/CHANGELOG.md#1600-2020-09-09

### CODE REVIEW

**Comment by RandomByte** [comment]
Support for Node.js 10 will end in less than a month: https://github.com/nodejs/Release#release-schedule

We should rather require Node.js 12

**Comment by RandomByte** [comment]
LGTM

File: azure-pipelines.yml [hunk 1]
```diff
@@ -8,20 +8,14 @@ trigger:
...
-    linux_node_8:
-      imageName: 'ubuntu-18.04'
-      node_version: 8.x
-    linux_node_10:
-      imageName: 'ubuntu-18.04'
-      node_version: 10.x
     linux_node_12:
-      imageName: 'ubuntu-18.04'
+      imageName: 'ubuntu-20.04'
       node_version: 12.x
```
Comment (matz3) [inline azure-pipelines.yml:13]: We shouldn't test with 12.x then, right?

**Review by matz3 — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (tobiasso85) [reply azure-pipelines.yml:13]: good point

**Review by matz3 — APPROVED** [review]

**Pull request merged**